Disher/View Controllers/RecipesViewController.m

```diff
@@ -141,6 +135,9 @@ - (void)pickerView:(UIPickerView *)pickerView didSelectRow:(NSInteger)row inComp
         [self refreshData];
     }
 }
+- (BOOL)textField:(UITextField *)textField shouldChangeCharactersInRange:(NSRange)range replacementString:(NSString *)string {
+    return NO;
+}
```

Comment:
is this needed?

Reply:
I am using this to prevent the user from entering text in the text field because I only want them to be able to use the picker view options